lib/erl_interface/configure.in

```diff
@@ -138,6 +138,7 @@ esac
 # Checks for libraries.
 AC_CHECK_LIB([nsl], [gethostbyname])
 AC_CHECK_LIB([socket], [getpeername])
+AC_CHECK_LIB([m], [isfinite])
```

Comment:
The existence of isfinite (and fallback on finite) is done differently in the rest of OTP.